src/javascript/binary/base/page.js

```diff
@@ -398,11 +398,13 @@ Header.prototype = {
     on_load: function() {
         this.show_or_hide_login_form();
         this.register_dynamic_links();
-        if (!this.clock_started) this.start_clock();
+        //start_clock_ws
```

Comment:
pointless comment. There's no need for a comment that says "start_clock_ws" if the next line says "if (!this.clock_started) this.start_clock_ws()"....